volatility/framework/plugins/linux/check_modules.py

```diff
@@ -0,0 +1,87 @@
+# Volatility
+
+# This file is Copyright 2020 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+"""
+@author:       Matt Tressler
```

Comment:
Hiya, we'd prefer not to have individual contact information in the files, it can lead to people being territorial over the code.  Hopefully the git log will be sufficient, but please let us know if that's not ok?